src/Streamly/Internal/Memory/Array/Types.hs

```diff
@@ -15,28 +15,33 @@ module Streamly.Internal.Memory.Array.Types
     (
       Array (..)
 
+    -- * Freezing and Thawing
+    , unsafeFreeze
+    , unsafeFreezeWithShrink
+    , unsafeThaw
+
     -- * Construction
-    , withNewArray
-    , newArray
-    , unsafeSnoc
-    , snoc
-    , spliceWithDoubling
+--    , withNewArray
```

Comment:
Why are these commented? Why aren't they removed?

Reply:
No specific reason. I got into a habit of commenting code ove removing. In this case they should be removed.